lib/rules/web/admin_console/4.5/operator_hub.xyaml

```diff
@@ -279,4 +279,45 @@ switch_to_yaml_view:
     content: Create by manually entering YAML or JSON definitions
   action: click_radio_input
   action: check_page_contains
+check_all_namespace_installation_mode:
+  action:
+    if_param: recommended_ns
+    ref: check_all_namespace_installation_mode_with_recommended_ns
+  action:
+    if_not_param: recommended_ns
+    ref: check_all_namespace_installation_mode_without_recommended_ns
+check_specific_namespace_installation_mode:
+  action:
+    if_param: recommended_ns
+    ref: check_specific_namespace_installation_mode_with_recommended_ns
+  action:
+    if_not_param: recommended_ns
+    ref: check_specific_namespace_installation_mode_without_recommended_ns
+check_all_namespace_installation_mode_with_recommended_ns:
```

Comment:
I would say using `check_all_namespace_installation_mode_with_recommended_ns` and `check_all_namespace_installation_mode_without_recommended_ns` is much clearer and easier to read in feature compared with `check_specific_namespace_installation_mode`